src/pages/regulatory/index.js

```diff
@@ -381,44 +357,79 @@ const Regulatory = () => {
+                                            key={1}
+                                            weight="bold"
+                                            target="_blank"
+                                            href="/regulatory/Champion_Group_Ltd.pdf"
                                             rel="noopener noreferrer"
                                         />,
                                     ]}
                                 />
                             </Text>
-                            <Text mt="2rem" max_width="58.8rem">
+                            <Text max_width="58.8rem">
+                                {localize(
+                                    'Clients in the rest of the world (except for restricted countries such as the USA, Canada, and Hong Kong) who wish to trade CFDs on financial instruments can have MetaTrader 5 accounts under Deriv (V) Ltd and Champion Group Ltd.',
+                                )}
+                            </Text>
+                        </CssGridColumn>
+
+                        <ResponsiveGrid align="flex-start">
+                            <Mobile>
+                                <StyledHeader lh="4rem" as="h4" mb="1.6rem">
+                                    {localize('Deriv (SVG) LLC')}
+                                </StyledHeader>
+                            </Mobile>
+                            <SVG />
+                        </ResponsiveGrid>
+                        <CssGridColumn>
+                            <Desktop>
+                                <StyledHeader lh="4rem" as="h4">
+                                    {localize('Deriv (SVG) LLC')}
+                                </StyledHeader>
+                            </Desktop>
+                            <Text mt="0.8rem" mb="1.6rem" max_width="58.8rem">
                                 {localize(
-                                    'Clients from the rest of the world (except for restricted countries such as the USA, Canada, and territories such as the European Union) who wish to trade CFDs on forex and cryptocurrencies will have their MetaTrader 5 accounts opened under Deriv (FX) Ltd.',
+                                    'Deriv (SVG) LLC, Hinds Buildings, Kingstown, St. Vincent and the Grenadines (company number 273 LLC 2020).',
+                                )}
+                            </Text>
+                            <Text max_width="58.8rem">
+                                {localize(
+                                    'Clients from the rest of the world (except for certain countries such as the USA, Canada, and Hong Kong) will have their accounts opened under Deriv (SVG) LLC.',
                                 )}
                             </Text>
                         </CssGridColumn>
+
                         <ResponsiveGrid align="flex-start">
                             <Mobile>
                                 <StyledHeader lh="4rem" as="h4" mb="1.6rem">
```

Comment:
If i compare to zeplin, the line 449 

(registration no. 108101) is removed from the sentence

Reply:
From line 482 to 503, it has a slight difference within the example from zeplin, but I think the current version is better than the zeplin version, not sure which one we should follow

Reply:
I think it is better to stick to zeplin design